src/table/cells/handleKeyPress.js

```diff
@@ -0,0 +1,74 @@
+const navigationEffect = (tableRows, nextRow, nextCol) => {
```

Comment:
should probably be called something like `moveFocus`

Reply:
sounds good. sure.